samples/universal-react-node/src/server/index.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+const SSRCaching = require("electrode-react-ssr-caching");
+
+process.on("SIGINT", () => {
+  process.exit(0);
+});
+
+const config = require("electrode-confippet").config;
```

Comment:
don't access `electrode-confippet.config` so early.  This actually invokes code to prepare the config.  Access it where it's first needed.